site/docs/v1/tech/core-concepts/applications.adoc

```diff
@@ -494,6 +474,14 @@ When enabled the Login API will return refresh tokens. This is functionally equi
 [field]#Allow token refresh# [optional]#Optional#::
 When enabled a JWT may be refreshed using the JWT Refresh API. This is functionally equivalent to enabling the OAuth2 Refresh Grant.
 
+===== Access control lists settings
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[.api]
+[field]#Access control list# [optional]#Optional# [since]#Available since 1.30.0#::
+The IP access control list that will be applied to this application.
```

Comment:
Our own UI doesn't yet describe this very well, but this is only used as a "UI" access control. In particular this means that it will be used to restrict access to the themed pages which is all of the front end SSO stuff. 

Also, when this is configured, it will take precedence over the tenant value if configured.